dev/com.ibm.ws.webcontainer.cors/bnd.bnd

```diff
@@ -51,7 +51,9 @@ instrument.classesExcludes: com/ibm/ws/webcontainer/cors/internal/resources/*.cl
 	com.ibm.ws.kernel.service,\
 	com.ibm.ws.webcontainer;version=latest,\
 	com.ibm.ws.org.osgi.annotation.versioning;version=latest,\
-	com.ibm.ws.transport.http
+	com.ibm.ws.transport.http,\
```

Comment:
Why are the changes to cors necessary?

Reply:
Unittests in cors use com.ibm.ws.transport.http code which now contains beta checks.   These beta classes weren't part of the classpath and caused ClassNotFoundExceptions

Reply:
ok can you create an issue to clean this up once we're not beta any longer?

Reply:
Yep: https://github.com/OpenLiberty/open-liberty/issues/28070